rest/import_test.go

```diff
@@ -646,7 +646,7 @@ func TestXattrImportMultipleActorOnDemandPut(t *testing.T) {
 	putResponse := rt.SendAdminRequest("PUT", fmt.Sprintf("/db/%s?rev=%s", mobileKey, revId), `{"updated":true}`)
 	goassert.Equals(t, putResponse.Code, 201)
 	base.JSONUnmarshal(putResponse.Body.Bytes(), &body)
-	log.Printf("Put response details: %s", putResponse.Body.Bytes())
+	log.Printf("PutWithBody response details: %s", putResponse.Body.Bytes())
```

Comment:
The rename caught other stuff

```suggestion
	log.Printf("Put response details: %s", putResponse.Body.Bytes())
```